core/src/main/scala/zio/logging/Logging.scala

```diff
@@ -36,6 +37,12 @@ object Logging {
       } yield ()
     )
 
+  def error(cause: Cause[Any]): ZIO[Logging, Nothing, Unit] =
```

Comment:
Would you mind sorting these methods alphabetically, here and in subclasses? 🙏

Reply:
done